src/format/asciidoc/format-asciidoc.ts

```diff
@@ -240,6 +256,51 @@ const asciidocBookExtension = {
   },
 };
 
+function project2Atlas(projDir: string, context: ProjectContext) {
+  // Cook up an atlas.json file that will be used as a placeholder
+  const bookStem = bookOutputStem(projDir, context.config);
+  const bookTitle = bookConfig("title", context.config);
+  const atlasJson = {
```

Comment:
Would this be cleaner as an externally loaded JSON file that you then do a few replacements on?  I could go either way, but I thought I would just suggest.

Reply:
It would be easy to change to that if needed, but I don't see huge upside. I could see in the long run this being replace with a project scaffold option for books that automatically generates the atlas json with additional options (based upon  user input). We can replace this then!